feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SessionDetailFragment.kt

```diff
@@ -92,7 +92,27 @@ class SessionDetailFragment : DaggerFragment() {
             }
 
         binding.bottomAppBar.setOnMenuItemClickListener {
-            handleNavigation(it.itemId)
+            when (it.itemId) {
+                R.id.session_share -> {
+                    // do something
+                }
+                R.id.floormap -> {
+                    handleNavigation(it.itemId)
+                }
+                R.id.session_calendar -> {
+                    val session = binding.session ?: return@setOnMenuItemClickListener false
```

Comment:
`true` sounds better and consume a click event if returning a result somehow.

Reply:
updated with this issue e4e55ca